js/views/itemEditVw.js

```diff
@@ -419,7 +463,7 @@ module.exports = Backbone.View.extend({
     //add formChecked class to form so invalid fields are styled as invalid
     this.$el.find('#contractForm').addClass('formChecked');
 
-    if(submitForm.checkValidity()){
+    if(this.checkFormValidity()){
```

Comment:
I needed to update the save to call a custom function which includes our custom description field validation in addition to the generic html5 checkValidity. The reason for that was because we only had the validation on the blur handler, so if you didn't ever focus in the description field and saved, validation wasn't being done on that field.